boards/xtensa/esp_wrover_kit/esp_wrover_kit.dts

```diff
@@ -88,6 +88,19 @@
 
 &gpio0 {
 	status = "okay";
+
+	rgb-leds-off {
+		gpio-hog;
+		gpios = <4 GPIO_ACTIVE_HIGH>, <2 GPIO_ACTIVE_HIGH>,
+			<0 GPIO_ACTIVE_HIGH>;
+		output-low;
+	};
```

Comment:
These are already in `gpio-leds`, how about just dropping it?

Reply:
they may be used later in the app, right?

Reply:
yeah but then the app should reinitialize them anyway or use the leds subsys

Reply:
so then I guess we should drop LEDs config, and just leave LCD, right?

Reply:
yeah I think it'd be reasonable